src/actions/guides/http_and_routing/flash.cr

```diff
@@ -34,6 +34,41 @@ class Guides::HttpAndRouting::Flash < GuideAction
     You can modify the layout or the component to add HTML classes, change where
     flash is rendered, etc.
 
+    ### Persisting flash messages through redirect
```

Comment:
I think maybe this section can just go at the bottom. The other two sections are pretty closely related on how to set your flash as where this section is just utilizing different parts of flash.